CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.9.4
```

Comment:
Mark this as 1.9.4-dev – also do the same in pubspec.yaml!

@natebosch – or would we consider this `1.10` since it adds a feature?

Reply:
I think `1.9.4-dev` is fine.

Reply:
updated. thank you.